src/test/java/org/folio/rest/impl/EntitiesCrudTest.java

```diff
@@ -0,0 +1,209 @@
+package org.folio.rest.impl;
+
+import java.net.MalformedURLException;
+import java.util.stream.Stream;
+
+import org.folio.rest.utils.TestEntities;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.EnumSource;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import io.restassured.response.Response;
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+class EntitiesCrudTest extends TestBase {
+
+  private final Logger logger = LoggerFactory.getLogger(EntitiesCrudTest.class);
+  private String sample = null;
+
+  static Stream<TestEntities> deleteOrder() {
+    return Stream.of(
+      TestEntities.ACQUISITIONS_UNIT_ASSIGNMENTS,
+      TestEntities.VOUCHER_LINES,
+      TestEntities.VOUCHER,
+      TestEntities.INVOICE_LINES,
+      TestEntities.INVOICE);
+  }
+
+  static Stream<TestEntities> deleteFailOrder() {
+    return Stream.of(
+      TestEntities.VOUCHER,
+      TestEntities.INVOICE);
+  }
+
+  static Stream<TestEntities> createFailOrder() {
+    return Stream.of(
+      TestEntities.ACQUISITIONS_UNIT_ASSIGNMENTS,
+      TestEntities.VOUCHER_LINES,
+      TestEntities.VOUCHER,
+      TestEntities.INVOICE_LINES);
+  }
+
+  @ParameterizedTest
+  @Order(1)
+  @EnumSource(TestEntities.class)
+  void testVerifyCollection(TestEntities testEntity) throws MalformedURLException {
+    logger.info(String.format("--- mod-orders-storage %s test: Verifying database's initial state ... ", testEntity.name()));
```

Comment:
Please remove or replace `mod-orders-storage` by `mod-invoice-storage`. This is applicable for all log messages in this class

Reply:
fixed